backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGLES20.java

```diff
@@ -263,7 +262,55 @@ public native void glCopyTexImage2D (int target, int level, int internalformat,
 
 	public native void glStencilOpSeparate (int face, int fail, int zfail, int zpass);
 
-	public native void glTexImage2D (int target, int level, int internalformat, int width, int height, int border, int format,
+	private static Buffer convert16bitBufferToRGBA8888 (Buffer buffer, int type) {
+		// TODO: 20.10.22 Can it be a different buffer type?
```

Comment:
Might be better suited to take this out to some util

Reply:
Do you have a suggestion of a already existing util class?

Reply:
BufferUtils is probably the closest one, but this seems a bit too niche for it to be there. I guess its probably ok to have here, but it would need to be protected so GLES30 can call it too where it needs to

Reply:
Done